admin/.env.example

```
@@ -1,19 +1,19 @@
 # copy to .env.local for development
+
+# === 必須 ===
+DATABASE_URL="postgresql://postgres:postgres@127.0.0.1:54332/postgres"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)